src/qnngds/devices/nanowire.py

```diff
@@ -77,13 +82,113 @@ def variable_length(
     constriction = NANOWIRE << line
     gnd = NANOWIRE << wire
     source.connect(source.ports[1], constriction.ports["top"])
-    constriction.connect(constriction.ports["bottom"], gnd.ports[1])
-
+    gnd.connect( gnd.ports[1], constriction.ports["bottom"])
+
+    NANOWIRE.rotate(rotation)
+    #NANOWIRE = pg.union(NANOWIRE, layer=layer)
+    if four_point_probe and rotation == -90:
+        add_voltage_probe(NANOWIRE, channel_w)
+    
+        nw_padplace = PadPlacement(
+        cell_scaling_factor_x= 2.5,
+        num_pads_n=2,
+        num_pads_s=2,
+        port_map_x={
+            1: ("N", 2),
+            2: ("S", 1),
+            3: ("N", 1),
+            4: ("S", 2)
+        },
+        probe_tip=WireBond()
+        
+        )
+
+        '''turn = pg.optimal_90deg(source_w)
```

Comment:
this commented out bit should probably just get deleted

Reply:
Yeah, I had left it in because I was trying to get the nanowires to work with a geometry rotated by 90deg but I was not quite able to get there...but since it's saved in some commits at least, I can always find it to pick that up again if I want to